tests/test_tiingo.py

```diff
@@ -98,6 +98,34 @@ def test_ticker_price_with_csv(self):
         rows = list(reader)
         assert len(rows) > 2  # more than 1 day of data
 
+    @vcr.use_cassette('tests/fixtures/ticker_price_with_volume_column.yaml')
+    def test_ticker_price_with_volume_column(self):
```

Comment:
Good descriptive test names 👍

Reply:
I saw I sloppy pasted the description of the tests itself, fixed it in the latest commit :)